### PR TITLE
fix: check if we are sent 512 characters or more

### PR DESCRIPTION
**Pull request by ppiques**
huge pr take your time to check all files and read it all

### CODE REVIEW

**Comment by theovgl** [comment]
> huge pr take your time to check all files and read it all

ahahah menteur

**Review by theovgl — APPROVED** [review]
Mais du coup on le fait quand meme ? Juzain et tout avait pas de limite non ?

File: src/Server.cpp [hunk 1]
```diff
@@ -106,6 +106,11 @@ int	Server::_handle_message(epoll_event &event)
 	else
 	{
 		buffer[bytes_read] = '\0';
+		if (peer.getMessage().length() + bytes_read > 512)
+		{
+			peer.clearMessage();
+			return 1;
+		}
```
Comment (vfurmane) [inline src/Server.cpp:113]: A debug message would be nice

File: src/Server.cpp [hunk 2]
```diff
@@ -106,6 +106,11 @@ int	Server::_handle_message(epoll_event &event)
 	else
 	{
 		buffer[bytes_read] = '\0';
+		if (peer.getMessage().length() + bytes_read > 512)
```
Comment (vfurmane) [inline src/Server.cpp:109]: The 512 characters limit could be a define (yes a define, not a static const)

**Review by vfurmane — CHANGES_REQUESTED** [review]

**Review by vfurmane — APPROVED** [review]

**Pull request merged**